src/cmd-buildextend-live

```diff
@@ -415,6 +415,10 @@ def generate_iso():
         # can be EFI/fedora or EFI/redhat
         grubpath = glob.glob(os.path.join(tmpisoroot, 'EFI/*/grub.cfg'))
         shutil.move(grubpath[0], os.path.join(tmpisoroot, 'boot/grub/grub.cfg'))
+        for key in list(files_with_karg_embed_areas):
+            if re.search(r"/EFI/.*/grub.cfg", key):
+                files_with_karg_embed_areas['/srv/fcos/tmp/buildpost-live/live/boot/grub/grub.cfg'] = files_with_karg_embed_areas[key]
```

Comment:
This bit looks wrong, we need to use relative paths here because not all build setups will use `/srv/fcos`.

Reply:
Yes, you are right I fixed it